xbrowse_server/matchmaker/utilities.py

```diff
@@ -233,6 +242,26 @@ def generate_slack_notification_for_incoming_match(response_from_matchbox,incomi
         message += 'These matches were sent back today (' + time.strftime('%d, %b %Y')  + ').'
         if settings.SLACK_TOKEN is not None:
             post_in_slack(message,settings.MME_SLACK_MATCH_NOTIFICATION_CHANNEL)
+        if settings.EMAIL_NOTIFICATIONS_ACTIVATED:
+            email_content = render_to_string(
+                'emails/mme_returned_match_result_message.txt',
+                {'query_institution': institution,
+                 'number_of_results': len(results_from_matchbox),
+                 'incoming_query_contact_genes':','.join(incoming_query_genes),
+                 'incoming_query_contact_phenotypes':', '.join([key + '(' + incoming_query_phenotypes[key]['name'] +')' for key in incoming_query_phenotypes]),
+                 'incoming_query_contact_url':incoming_patient_as_json['patient']['contact'].get('href','(sorry I was not able to read the information given for URL'),
+                 'incoming_query_contact_name':incoming_patient_as_json['patient']['contact'].get('name','(sorry I was not able to read the information given for name'),
+                 'match_results':'\n'.join(match_results),
+                 'email_addresses_alert_sent_to':','.join([i for i in seqr_project.mme_contact_url.replace('mailto:','').split(',')]),
+                 },
+            )
+            send_mail('match found by matchbox, the Matchmaker Exchange @Broad', 
+                      email_content, 
+                      settings.FROM_EMAIL, 
+                      #commenting for now for advanced testing
+                      #[i for i in seqr_project.mme_contact_url.replace('mailto:','').split(',')],
+                      ['harindra@broadinstitute.org'],
```

Comment:
hey Ben, I think you missed the 2 comments just above this piece of code. The line above commented is the eventual real code here. I didn't want that to be production immediately till was sure everything was OK after we use this in production for a bit (the risk of emailing out gene names to wrong people). As you can see the real code  (commented just above the line) uses the model that contains information entered by the submitter on who should be alerted.

Though for this first release, I could use settings.ADMINS sure, since it's just you and me probably still?

Reply:
ah I see, that makes sense.